src/model/Question.java

```diff
@@ -141,35 +141,29 @@ public void setCategoryPathName(int path) {
 
      }
 
-    /**This method randomly generates a question ID number*/
+    /**This method randomly generates a question ID number.*/
      public int selectQuestion(){
          Random random = new Random();
 
          if (categoryID == 1) { //Geek Out
-             range = (int )(Math.random() * (120 - 101) + 101);
-             System.out.println("hi1 " +range);
+             randomID = (int )(Math.random() * (120 - 101) + 101);
```

Comment:
`randomID = rand.nextInt(120) + 101;`
can alternatively do the job here. Kinda looks good, in my eyes!